### PR TITLE
LRAC-7581-pt3

### PR DESCRIPTION
**Pull request by joshitagaki**
https://issues.liferay.com/browse/LRAC-7581

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSegments.path [hunk 1]
```diff
@@ -34,6 +39,21 @@
     <td>//div[contains(@class,'dropdown')]/button[contains(@class,'item')]</td>
     <td></td>
 </tr>
+<tr>
+    <td>DELETED_CRITERIA</td>
+    <td>//div[@class='criteria-message']td>
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSegments.path:44]: hey @joshitagaki , it is missing '<' when closing the td tag.

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSegments.path [hunk 2]
```diff
@@ -34,6 +39,21 @@
...
+    <td></td>
+</tr>
+<tr>
+    <td>DUPLICATED_CRITERIA</td>
+    <td>(//div[@class='criteria-statement']//div[text()='${criteriaName}'])[${totalCriteriaCount}]td>
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSegments.path:49]: hey @joshitagaki , it is missing '<' when closing the td tag.

**Review by alexcandidos — COMMENTED** [review]
hey @joshitagaki , the code looks good to me! I just saw some td's tags not closed. ;)

**Comment by alexcandidos** [comment]
hey @joshitagaki , the code looks good to me!

**Comment by joshitagaki** [comment]
ci:forward

**Comment by joshitagaki** [comment]
ci:test:sf

**Comment by joshitagaki** [comment]
ci:forward

**Pull request closed without merging**